packages/block-editor/src/components/block-mobile-toolbar/block-actions-menu.native.js

```diff
@@ -175,6 +182,21 @@ const BlockActionsMenu = ( {
 				);
 			},
 		},
+		convertToRegularBlocks: {
+			id: 'convertToRegularBlocksOption',
+			label: __( 'Convert to regular blocks' ),
+			value: 'convertToRegularBlocksOption',
+			onSelect: () => {
+				createSuccessNotice(
+					sprintf(
+						/* translators: %s: name of the reusable block */
+						__( '%s converted to regular blocks' ),
+						reusableBlock?.title?.raw || blockTitle
```

Comment:
If the reusable block's title is not defined it will fallback to "Reusable block".

Reply:
Additionally I realised that the `notice` component doesn't handle content overflow (see attached screenshot).

Block selected|Block converted to regular blocks
--|--
<img src=https://user-images.githubusercontent.com/14905380/115539779-acf16b00-a29d-11eb-9ae8-d3017684d7d8.png width=400>|<img src=https://user-images.githubusercontent.com/14905380/115540151-15404c80-a29e-11eb-93e1-cda931c231b1.png width=400>

Reply:
I created the fix in this [PR](https://github.com/WordPress/gutenberg/pull/31083).